src/teaching-element/Assessment/FillBlank.vue

```diff
@@ -43,6 +44,9 @@ export default {
     retake(val) {
       if (val) this.userAnswer = [];
     },
+    submission(val) {
+      if (val) this.userAnswer = val;
```

Comment:
Just my train of thought:
This behavior is different from the one that will happen on `created` ( `userAnswer: this.submission || [] ` ).
If `val` happens to change to `null` you will persist the old ones which seems weird to me?
Maybe do something like this:
```js
this. userAnswer = val || [];
```

Reply:
Good catch.